docs/concepts/glossary.md

```diff
@@ -298,6 +298,10 @@ In IPFS and [IPLD](#ipld), a _link_ usually means a pointer to some [CID](#cid).
 
 ## M
 
+### IPFS Mainnet
+
+IPFS Mainnet is a term used to describe the default or "main" network that many IPFS implementations connect to. Mainnet is a subset of all the [IPFS implementations](https://specs.ipfs.tech/architecture/principles/#ipfs-implementation-requirements) and typically includes implementations that connect to the [Amino DHT](#amino) and support [Bitswap](#bitswap), [UnixFS](#unixfs), and the [IPFS Gateway](#gateway). This has mostly been assumed for the IPFS network and is not emphasized for anyone other than protocol experts. Nonetheless, IPFS Mainnet is a useful distinction in a world of many [IPFS implementations](../concepts/implementations.md) with varying degrees of interoperability.
```

Comment:
For me, "mainnet" is less about implementations than configs, i.e. a private-network kubo is not on mainnet, but a default-config kubo puts everything it `ipfs add`s onto mainnet, if you catch my drift.  I could be wrong, tho!

```suggestion
IPFS Mainnet is a term used to describe the default or "main" network that most IPFS implementations connect to by default configuration.
Most [IPFS implementations](https://specs.ipfs.tech/architecture/principles/#ipfs-implementation-requirements) were designed to work with mainnet, but many can be configured instead to use private networks (i.e. only sync with manually allowlisted peers) or only serve content generated locally.  

Mainnet as a network and data substrate refers to the [Amino DHT](#amino). comprised of nodes that synchronize their contents using the [Bitswap](#bitswap) sync protocol;
as all data is, by default, converted to [UnixFS](#unixfs) at ingress by these implementations, and [IPFS Gateways](#gateway) are configured to assume [UnixFS](#unixfs)-only data, other encodings are sometimes considered incompatible with or problematic for use with bitswap, and as such, not "Mainnet-compatible".
Most IPFS documentation is written assuming all Mainnet-compatible configurations and network topology, because straying from these defaults can break many assumptions of the system and is recommended only for advanced applications.
For this reason, compatibility with IPFS Mainnet is often referred to as "full IPFS compatibility", and many [IPFS implementations](../concepts/implementations.md) require additional care to interoperate at the bitswap-sync, DHT-advertising, or CID-fetching layers.
```

(totally just winging it, feel free to take or leave any of that, but this would be my "interop maxi"/"roast my docs" take on the full meaning of Mainnet!)

Reply:
I've integrated some of the information from your suggestion. For consistency with other glossary terms, I'm keeping this brief. If you feel it's necessary to add all the additional information, I would create a new page dedicated to this topic. But I'd consider that out of scope for this PR.